app/src/main/java/com/gh4a/fragment/BookmarkDragHelperCallback.java

```diff
@@ -0,0 +1,36 @@
+package com.gh4a.fragment;
```

Comment:
This should probably reside in a different package or (preferably?) in BookmarkListFragment itself.